python/graphstorm/gconstruct/file_io.py

```diff
@@ -30,6 +30,64 @@
 
 from .utils import HDF5Handle, HDF5Array
 
+
+def read_index(data_file, column=None):
+    """ Read the index from a JSON/parquet file.
+
+    Each row is a JSON object that contains an index.
+
+    Parameters
+    ----------
+    data_file : str
```

Comment:
Please note: we should all there are a list of parquet files instead of just one.
And make sure we sort the files before loading them.